src/camera.cpp

```diff
@@ -211,10 +211,12 @@ static inline v2f dir(const v2f &pos_dist)
 	return v2f(std::fabs(x), std::fabs(y));
 }
 
-void Camera::addArmInertia(f32 player_yaw, f32 frametime)
+void Camera::addArmInertia(f32 player_yaw)
 {
-	m_cam_vel.X = std::fabs((m_last_cam_pos.X - player_yaw) / frametime) * 0.01f;
-	m_cam_vel.Y = std::fabs((m_last_cam_pos.Y - m_camera_direction.Y) / frametime);
+	m_cam_vel.X = std::fabs((m_last_cam_pos.X - player_yaw) / 0.016f) * 0.01f;
+	m_cam_vel.Y = std::fabs((m_last_cam_pos.Y - m_camera_direction.Y) / 0.016f);
+	f32 gap_X = std::fabs(55.0f - m_wieldmesh_offset.X);
```

Comment:
maybe add macros for 55.0f and -35.0f to define textualy what it is (it's used in two places)